logstash-core/src/test/java/org/logstash/common/io/RecordIOWriterTest.java

```diff
@@ -99,6 +99,28 @@ public void testReadWhileWrite() throws Exception {
         reader.close();
     }
 
+    @Test
```

Comment:
I colocated it with a very similar (`test testReadWhileWrite`), but yeah - I think you are right, given that it verifies the fixing of a reader bug, not a writer bug